.github/workflows/test-action.yml

```diff
@@ -5,19 +5,18 @@ name: Smoke test the action
 
 on:
   workflow_call:
-    inputs:
-      runs-on:
-        required: true
-        type: string
 
 env:
   RUST_BACKTRACE: 1
-  CARGO_TERM_COLOR: always
 
 jobs:
   run-tests:
     name: Run tests
-    runs-on: ${{ inputs.runs-on }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        toolchain: ['1.66', 'stable', 'beta', 'nightly']
```

Comment:
Should we add `1.67` here? Current stable is `1.68`.

Reply:
Added! It's three more jobs in the CI, but I suppose it's not a problem?